docs/developer-guide/working-with-custom-formatters.md

```diff
@@ -254,48 +296,13 @@ error semi
 
 ### Using a JSON formatter first
 
-It is a bit more complicated, but using a simple formatter we can get the raw output to stdout
-
-```javascript
-// json.js
-module.exports = function ( results ) {
-    return JSON.stringify( results );
-}
-```
-
-And then the formatter can read from stdin
+You can use ESLint's built-in JSON formatter first, and pipe the output to another program that accepts JSON linter results.
```

Comment:
If we change the header of this section this could be `You can use ESLint's built-in JSON formatter and pipe the output to another program that accepts JSON linter results.`

Reply:
Yeah!  I hear you on the title being kinda off now.  I'll think about alternate titles for the section and get back to you, probably tomorrow.  Will also make that change in wording once the title is nailed down.  Thanks for reviewing!

Reply:
Ok, how about just "Using built-in formatters with custom formatters" ?  I guess you could use the other formatters too, like HTML or something, and then send it to another program to do something with.  Since this is the "using custom formatters" page, the "other program" is probably going to format the results.  LMK what you think or if you've got other ideas.  I'll hold off on submitting another PR until we come to a decision.

Reply:
Yeah, okay.  It's a little long but it's probably better to be too long than be confusing.  How about we try it out?  I'll change the title to that and resubmit the PR.

Reply:
Sounds good - we can always change it later if it's confusing! No need to resubmit - you can either add another commit to this branch (we squash and merge) or amend the current commit and force push it. Thanks for working on this!

Reply:
Cool!  I'll try to amend the current commit.  Thank you!

Reply:
I tried to amend the current commit, it looks like it made a new commit.  Is there anything else I need to do, or will this just get squashed and merged?  Thanks!

Reply:
Multiple commits is fine! We'll just squash when we merge. Thanks!

Reply:
Awesome!  Thank you!